DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/HostedElementConversionToSpeckle.cs

```diff
@@ -24,10 +25,12 @@ ISpeckleConverterToSpeckle converter
 
   public List<Base> GetHostedElementsConverted(Element host)
   {
-    var convertedHostedElements = new List<Base>();
-
-    var hostedElementIds = GetHostedElementIds(host);
+    return GetHostedElementsConvertedFromIds(host, host.GetHostedElementIds());
```

Comment:
Hosted elements were kind of OOS for Alpha so maybe this is fine for now but I think @clairekuang needs to be across this for other object relationships such as blocks, instancing etc...

Maybe add a POC: comment for now to the effect of, needs to be reviewed